skopt/searchcv.py

```diff
@@ -398,6 +399,146 @@ def add_spaces(self, names, search_spaces):
         for space, name in zip(search_spaces, names):
             self.search_spaces_[name] = space
 
+    # copied for compatibility with 0.19 sklearn from 0.18 BaseSearchCV
+    @property
+    def best_score_(self):
```

Comment:
You are right we need to set them by hand if we copy the 0.18 code, in 0.19 they get set by `fit()`.